### ISSUE TITLE
Please release updated RStudio

### ISSUE THREAD

**Issue by dhirschfeld** [ContinuumIO/anaconda-issues#11102]
### Actual Behavior
As mentioned on [gitter](https://gitter.im/conda/conda?at=5d1c362bf0f22f66450575c5) I get an UnsatisfiableError when trying to install RStudio with the latest `r_base`

It would be great if an updated RStudio which depends on the latest (released) qt could be released

**Comment by mingwandroid** [comment]
Thanks for bring this up @dhirschfeld. RStudio is a *big* undertaking (but one I'd dearly love to address). The main issue is that we need to update Qt beyond 5.9 on all platforms.

I'd like RStudio recipe maintainence to move to conda-forge eventually too, do you think there's enough interest in tackling that?

Pinging @msarahan as it's very important since it's on Anaconda Navigator (and a great IDE of course that's now badly out of date in Anaconda Distro!)

**Comment by dhirschfeld** [comment]
Yeah, I understand updating qt is a big undertaking. I'd be perfectly fine with it being in conda-forge but I'm not sure those with enough expertise are that interested in R? :/ 

The latest `rstudio`:
```
rstudio                      1.1.456  vc14h1ad3211_1  pkgs/r
```
...depends on `qt 5.6.*` whereas the latest released `qt` is 5.9.7. Since all new libraries which depend on `qt` require the newer version it makes it hard to integrate into a cutting edge analytics stack.

I think that rebuilding rstudio against the current 5.9.7 might resolve the immediate problem but then I'm not sure if the latest rstudio requires an even newer qt?

I think there may be interest on conda-forge to update to `qt=5.12` as both @ocefpaf and @isuruf were  involved in the update to 5.9. I  think to get it over the line will still require a decent amount of your time  @mingwandroid as the people with that expertise are few and far between.

Hopefully if `qt` does get updated then updating `rstudio` may not be that much extra work on top.

We aren't heavily invested in R but we do have some analysts who use it and insofar as is possible I'd like to provide them with a functional development environment which means deploying RStudio (which *is* a very nice IDE). Where possible I prefer to deploy all our tools and libraries using conda so it would be great if (a compatible) RStudio was available with conda too.

**Comment by mingwandroid** [comment]
How about a somewhat horrible measure of doing a binary repackaging one-off here? So long as the name ends with `bin` or `binary` and maybe has a `-` before that and I'm all happy that we'd not clash in package namespace. I wonder if conda-forge would be interested in this, but it does feel like capitulation somewhat! Still, people need to get stuff done and RStudio do do a good job so maybe that's the pragmatic approach for now?

**Comment by dhirschfeld** [comment]
I think that's a good interim work-around as there are other repackaged binaries on CF.

**Comment by mingwandroid** [comment]
`RSTUDIO_WHICH_R` is of interest.

**Comment by dhirschfeld** [comment]
> *`RSTUDIO_WHICH_R` is of interest.*

Thanks @mingwandroid - you just saved me a ton of time! I had been (unsuccessfully) googling exactly how to do just that.

**Comment by mingwandroid** [comment]
A pleasure!

**Comment by ocefpaf** [comment]
> How about a somewhat horrible measure of doing a binary repackaging one-off here? So long as the name ends with `bin` or `binary` and maybe has a `-` before that and I'm all happy that we'd not clash in package namespace. I wonder if conda-forge would be interested in this, but it does feel like capitulation somewhat! Still, people need to get stuff done and RStudio do do a good job so maybe that's the pragmatic approach for now?

While I do prefer a build from source I tried building `rstudio` and failed miserably. TL;DR I'm not against a binary repackage at this point.
